<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>29</y>
     <width>761</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="armButton">
        <property name="text">
         <string>Arm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="unknownButton">
        <property name="text">
         <string>Unkown</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="restButton">
        <property name="text">
         <string>Rest</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="flatButton">
        <property name="text">
         <string>Flat</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pouchButton">
        <property name="text">
         <string>Pouch</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bagButton">
        <property name="text">
         <string>Bag</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="boxButton">
        <property name="text">
         <string>Box</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
